sinaloa-test/nitro_ec2_terminate_root.sh

```diff
@@ -0,0 +1,5 @@
+INFO=$(aws ec2 describe-instances --filters "Name=tag:Veracruz,Values=RootEnclave"| jq -r '.Reservations[].Instances[].InstanceId')
```

Comment:
Can we have the standard licensing/copyright header added?

Reply:
...actually, some explanation in the header about what these script files are doing would be useful, too.